docker-compose.yml

```diff
@@ -10,12 +10,10 @@ services:
     depends_on: 
       - es
   es:
-    image: elasticsearch:7.7.1
+    image: elasticsearch:6.8.10
```

Comment:
why did you change elasticsearch to version 6.8.10 in place of 7.8.1?

Reply:
was having issues getting it to work with 7.8.1